isis/tests/FunctionalTestsFindFeatures.cpp

```diff
@@ -0,0 +1,216 @@
+#include "findfeatures.h"
+
+#include <QTemporaryFile>
+#include <QTextStream>
+#include <QStringList>
+
+#include "Fixtures.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "SurfacePoint.h"
+#include "ControlPoint.h"
+#include "Latitude.h"
+#include "Longitude.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using ::testing::HasSubstr;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/findfeatures.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestFindFeaturesDefault) {
+  // Setup output file
+  QVector<QString> args = {"algorithm=brisk/brisk",
+                           "match=" + tempDir.path() + "/cube3.cub",
+                           "fromlist=" + twoCubeListFile,
+                           "tolist=" + tempDir.path() + "/toList.txt",
+                           "tonotmatched=" + tempDir.path() + "/unmatched.txt",
+                           "maxpoints=5000",
+                           "epitolerance=1.0",
+                           "ratio=.65",
+                           "hmgtolerance=3.0",
+                           "onet=" + tempDir.path() + "/network.net",
+                           "networkid=new",
+                           "pointid=test_network_????",
+                           "description=new",
+                           "debug=false"};
+  UserInterface options(APP_XML, args);
+  Pvl log;
```

Comment:
Since we don't use the log data in the test and it can be null in the findfeatures call, can we remove this variable?